SHARDING.md

```diff
@@ -321,17 +321,25 @@ Following
 test the write path for the new shard. This will sign an entry with a private key and upload
 the signing event to the log.
 
-## WIP: Update Prober
+## Run prober tests
 
-We will need a mechanism for running prober tests against a new deployment before we've
-distributed an updated SigningConfig and TrustedRoot. Assuming this is a parameter to
-a GitHub Actions workflow, a deployer should provide the URL to the new shard and run
-prober tests.
+We can manually run prober tests using the URL and public key of the new shard before 
+their being distributed by TUF.
 
-Tracked in https://github.com/sigstore/rekor-tiles/issues/46. The prober will pick up
-new shards from the SigningConfig, with the latest active shard having no end date.
-Only the active shard will be used to test write traffic. All other shards in the SigningConfig
-will still support read traffic.
+For the [GitHub Actions prober](https://github.com/sigstore/sigstore-probers/actions/workflows/prober-staging.yml), 
+no configuration is necessary, as the URL and public key of the new shard can be provided as parameters to the workflow, 
+so that the new shard will be tested instead of any in the SigningConfig and TrustedRoot fetched from TUF. Without these 
+parameters, the prober will pick up new shards from the SigningConfig, with the latest active shard having no end date. 
+Only the active shard will be used to test write traffic. All other shards in the SigningConfig will still support read 
+traffic.
+
+The in-cluster prober, both in 
+[production](https://github.com/sigstore/public-good-instance/blob/e1c0302fdcde4212696abc3d3a5023a54a39f031/argocd/bootstrap-utilities/production/values.yaml) 
+and 
+[staging](https://github.com/sigstore/public-good-instance/blob/e1c0302fdcde4212696abc3d3a5023a54a39f031/argocd/bootstrap-utilities/staging/values.yaml), 
+uses a hardcoded SigningConfig and TrustedRoot, which must be updated to include the URL and public key of the new shard.
```

Comment:
Can you link to a specific line in the configurations as an example?

Reply:
Also, should we move this section for updating the in-cluster prober after where we've constructed the TUF-distributed files?